specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/examples/runCommandExamples/VirtualMachineRunCommand_Get.json

```diff
@@ -34,7 +34,9 @@
           "asyncExecution": false,
           "runAsUser": "user1",
           "timeoutInSeconds": 3600,
-          "provisioningState": "Succeeded"
+          "provisioningState": "Succeeded",
+          "outputBlobUri": "Base URI of a storage append blob(without SAS token)",
+          "errorBlobUri": "Base URI of a storage append blob(without SAS token)"
```

Comment:
could you please put some "real values" in the example files? We should already have descriptions on these properties, we do not need to duplicate that in the examples.
The example files should be showing what these properties really look like in a real traffic

Reply:
done